DFC.ServiceTaxonomy.CompUi/Handlers/EventGridHandler.cs

```diff
@@ -47,7 +41,7 @@ public async Task SendEventMessageAsync(Processing processing, ContentEventType
                             await _eventGridClient.Publish(new ContentEvent(eventData, ContentEventType.StaxDelete, StaxDeleteMessage + eventData.ContentType));
                             break;
                         default:
-                            _logger.LogError($"Current content could not be retrieved for Content Item ID: {processing.ContentItemId}");
+                            _logger.LogError($"Current content could not be retrieved for Content Item ID: {contentData.ContentItemId}");
```

Comment:
Is this message in the wrong place? Should it have been in an `else` for the `if (eventData != null)`?